tests/yam/migration/migration_unattended.pm

```diff
@@ -0,0 +1,58 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Migration activation then reboot to perform migration.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use grub_utils 'grub_test';
+use migration 'disable_installation_repos';
+use power_action_utils 'power_action';
+use utils 'zypper_call';
+
+sub await_migration {
+    my $timeout = 600;
+    my @tags = qw(migration-exit grub2);
+    while (1) {
+        my $ret = check_screen \@tags, 30;
+	$timeout -= 30;
+        diag("left total await_migration timeout: $timeout");
+	if (match_has_tag('grub2')) {
+            last;
+        }
+	if (match_has_tag('migration-exit')) {
```

Comment:
this name is confusing, migration doesn't exit, should be better 'migration-running'.